docs/codacy-api/examples/adding-people-to-codacy-programmatically.md

```diff
@@ -69,4 +69,4 @@ email4@codacy.com
 
 ## See also
 
--   [API tokens](../api-tokens.md)
```

Comment:
The other example page had a "See also" link to the API tokens because it was about creating API tokens programmatically. But here we can link to "Adding people instead".